### DISCUSSION TITLE
Указание путей плагинов при запуске far2l

### DISCUSSION THREAD

**Discussion by shmuz** [elfmz/far2l#1208] [Q&A] [answered]
Что-то у меня ключ `/p` не работает, ни сам по себе (должен запускать Фар без плагинов), ни с указанием пути.
Этой функциональности в far2l нет?

**Comment by elfmz** [comment, marked as answer]
Похоже да, функциональность потерялась при переносе и никто не заметил.. Но ее реализация несколько все усложнит, так как far2l при установке в линукс (ну, во многие линуксы) имеет свойство при `sudo make install` размазывать свое содержимое по различным директориям согласно https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard Вот тут было обсуждение на эту тему - https://github.com/elfmz/far2l/issues/84

**Comment by shmuz** [comment]
OK, спасибо. Может тогда из хелпа убрать?

**Reply by elfmz** [reply]
убрал, точнее закоментил - вдруг когданить руки дойдут..

**Comment by shmuz** [comment]
Как вариант, можно вернуть только ключ /p без параметров, запрещающий загрузку плагинов.
